### DISCUSSION TITLE
Freeze after restart

### DISCUSSION THREAD

**Discussion by FwOSnS3KcR4N6** [gickowtf/pixoo-homeassistant#71] [General]
After a restart I have to change and update my configuration every time. Otherwise no new data will be displayed. Am I the only one with the problem?

**Comment by SB2DD** [comment]
This is very possible. There has been a new ha core update that broke some integrations, so maybe it broke this one too. Do you mind sharing what's your ha core version?

**Reply by FwOSnS3KcR4N6** [reply]
Thanks, after update all the containers everything works fine.
